protolint.yaml

```diff
@@ -0,0 +1,8 @@
+lint:
+  rules:
+    remove:
+      - MESSAGE_NAMES_UPPER_CAMEL_CASE
+      - ENUM_FIELD_NAMES_ZERO_VALUE_END_WITH
+      - PACKAGE_NAME_LOWER_CASE
+      - REPEATED_FIELD_NAMES_PLURALIZED
+      - FIELD_NAMES_LOWER_SNAKE_CASE
```

Comment:
We should try to fix `MESSAGE_NAMES_UPPER_CAMEL_CASE`, `REPEATED_FIELD_NAMES_PLURALIZED` and `FIELD_NAMES_LOWER_SNAKE_CASE`.